app/views/notes/rss.rss.builder

```diff
@@ -9,11 +9,10 @@ xml.rss :version => "2.0" do
 
      body = node.latest.render_body
      body = "<p><![CDATA[ <img src='"+node.main_images.path(:default)+"' alt='"+node.main_image.title+"' > ]]></p> " + node.body if node.main_image
-     author = node.author.mail+" ("+node.author.name+")"
 
      xml.item do
        xml.title       node.title
-       xml.author      author
+       xml.author      node.author.user.has_power_tag('twitter') ? "@" + node.author.user.get_value_of_power_tag('twitter') : node.author.username
```

Comment:
@jywarren `node.author` did not have `has_power_tag` or `get_value_of_power_tag` , I had to get call the `user` on the `author` to call it. Is it ok ?